src/main/java/mate/controller/DeleteDriverController.java

```diff
@@ -0,0 +1,22 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.DriverService;
+
+public class DeleteDriverController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        driverService.delete(Long.parseLong(req.getParameter("id")));
+        resp.sendRedirect("/drivers");
```

Comment:
Add context path to the redirection
![image](https://user-images.githubusercontent.com/32339836/230405650-386cba13-9520-4945-b1bd-cea90b4879ed.png)